docs/source/openstack.rst

```diff
@@ -197,5 +201,4 @@ Alternatively, the credentials path can be set as an environment variable,
 .. code-block:: bash
 
    $ export CREDS_PATH="/path/to/credential_dir/"
-   $ linchpin -v up
-
+   $ linchpin -v
```

Comment:
is this correct?  why is up dropped?

Reply:
I was having some editor issues. I must have accidentally deleted that